data/FR.france/2007-2013/validate_source.py

```diff
@@ -0,0 +1,9 @@
+from datapackage_pipelines.wrapper import ingest, spew
+import json
+import os
+
+print(os.getcwd())
+datapackage_file, _, _ = ingest()
+datapackage = json.loads(datapackage_file)
```

Comment:
This and the next file are unfinished work that should not be in this PR. This processor has been rewritten replaced by a common processor.